pkg/devfile/generator/generators.go

```diff
@@ -238,6 +238,7 @@ func GetDeployment(devfileObj parser.DevfileObj, deployParams DeploymentParams)
 // PodTemplateParams is a struct that contains the required data to create a podtemplatespec object
 type PodTemplateParams struct {
 	ObjectMeta metav1.ObjectMeta
+	options    common.DevfileOptions
```

Comment:
Is `options` only intended to be used within the generator package and that's why it's private?

Reply:
No, it should be public. Thanks for the catch